code/modules/food_and_drinks/restaurant/generic_venues.dm

```diff
@@ -22,8 +22,13 @@
 	var/obj/item/object_to_order = customer_data.get_order(src)
 
 	. = object_to_order
-
-	customer_pawn.say(order_food_line(object_to_order))
+	var/line_to_say = order_food_line(object_to_order)
+	if(prob(1) || SSevents.holidays && SSevents.holidays[APRIL_FOOLS])
+		line_to_say = "Yeah can I get a mothafuckin uh"
+		while(prob(90))
```

Comment:
90% chance of lag with my burger thanks

Reply:
sorry we serve CHECK_TICK here

Reply:
Just do `for(var/i in 1 to rand(min, max))`

Reply:
> 
> 
> sorry we serve CHECK_TICK here

please god no I promise this will break somehow

Reply:
fixed